l2-contracts/contracts/bridge/L2ERC20Bridge.sol

```diff
@@ -116,6 +161,25 @@ contract L2ERC20Bridge is IL2Bridge, Initializable {
         emit WithdrawalInitiated(msg.sender, _l1Receiver, _l2Token, _amount);
     }
 
+    /// @notice Initiates a withdrawal by burning merge funds on the contract and sending the message to L1
+    /// where tokens would be unlocked
+    /// @param _l1Receiver The account address that should receive funds on L1
+    /// @param _l2Token The L2 token address which is withdrawn
+    /// @param _amount The total amount of tokens to be withdrawn
+    function withdrawFromMerge(address _l1Receiver, address _l2Token, uint256 _amount) external {
+        IERC20Upgradeable(_l2Token).safeTransferFrom(msg.sender, address(this), _amount);
```

Comment:
get _l2MergeToken from portal
IERC20Upgradeable(_l2MergeToken).safeTransferFrom(msg.sender, address(this), _amount);